packages/design-system/src/components/NativeDialog/NativeDialog.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useRef, useEffect, useLayoutEffect } from 'react';
+import dialogPolyfill from './polyfill';
+
+interface NativeDialogProps {
+  children: React.ReactNode;
+  /**
+   * Function called to close dialog.
+   */
+  exit: any;
```

Comment:
Could maybe be a little more strict? In a few other places we use:

`(...args: any[]) => any`